admin/tests/acceptance/authenticated/sessions/list/to-be-published_test.js

```diff
@@ -127,7 +127,7 @@ module('Acceptance | authenticated/sessions/list/to be published', function (hoo
       });
 
       test('it should be clickable if current user is Certif', async function (assert) {
-        assert.expect(3);
+        assert.expect(2);
```

Comment:
curieuse de savoir ce que cet expect fait ici :o

Reply:
c'est parceque les expects sont dans des functions. du coup il demande que tu valides des expect dans ton it dans tout les cas.